docs/Makefile

```
@@ -37,7 +37,7 @@ CLEAN_DIR	  = $(shell pwd)/__build__
 HOSTNAME      = localhost
 
 # Dependencies for building documentation
-DEPENDENCIES = robotframework Sphinx sphinxcontrib-napoleon \
+DEPENDENCIES = robotframework "Sphinx>=3,<4" sphinxcontrib-napoleon \
```

I think better to use exact version which we confirmed(used). technically this requirement will meet even with 3.0.0.
```suggestion
DEPENDENCIES = robotframework "Sphinx==3.5.4" sphinxcontrib-napoleon \
```